ftw/htmlblock/browser/htmlblock.py

```diff
@@ -0,0 +1,6 @@
+from zope.publisher.browser import BrowserView
+
+
+class HtmlBlockView(BrowserView):
+    def __call__(self):
+        return self.index()
```

Comment:
I think this is default behavior (the call method of BrowserView does the same), therefore superfluous.
Actually you could probably not have a class at all for the view and just use the page template 😉